documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+
+When using the operator, each deployed domain is specified by a domain resource that you define which describes important aspects of the domain. These include the location of the WebLogic Server image you wish to use, a unique identifier for the domain called the `domain-uid`, any PVs or PVC the domain pods will need to mount, the WebLogic clusters and servers which you want to be running, and the location of its domain home.
+
+Multiple deployments of the same domain are supported by specifying a unique `domain-uid` string for each deployed domain and specifying a different domain resource. The `domain-uid` is in turn used by the operator as the name-prefix and/or label for the domain's Kubernetes resources that the operator deploys for you. The WebLogic configuration of a domain's deployments optionally can by customized by specifying configuration overrides in the domain resource -- which, for example, is useful for overriding the configuration of a data source URL, user name, or password.
+
+The operator does not specify how a WebLogic domain home configuration is created. You can use WLST, REST, or a very convenient new tool called [WebLogic Server Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling) (WDT). WDT allows you to compactly specify WebLogic configuration and deployments (including JMS, data sources, applications, authenticators, and such) using a YAML file and a ZIP file (which include the binaries). The operator [samples]({{< relref "/samples/simple/_index.md" >}}) show how to create domains using WLST and using WDT.
+
+
+**Q:** Is the Administration Server required? Node Manager?
+
+**A:** Certification of both WebLogic on Docker and WebLogic in Kubernetes consists of a WebLogic domain with the Administration Server. The operator configures and runs Node Managers for you within a domain's pods - you don't need to configure them yourself - so their presence is largely transparent.
+
+***
+
+#### Communications
+
+**Q:** How is location transparency achieved and the communication between WLS instances handled? T3s?
+
+**A:** Inside the Kubernetes cluster, we use the cluster IP (acts as a DNS name) and ingress controller, which allows for the pods with WebLogic Servers to move to different nodes in the Kubernetes cluster and continue communicating with other servers.  
+
+For T3 communication outside the Kubernetes cluster, we use NodePort and configure a WebLogic channel for T3 RMI.  See the blog, [T3 RMI Communication for WebLogic Server Running on Kubernetes](https://blogs.oracle.com/weblogicserver/t3-rmi-communication-for-weblogic-server-running-on-kubernetes), which explains RMI T3 communication in detail. All QoS are supported for T3 and HTTP communication (SSL, administration port).
```

Comment:
The above is out of date.  How about:  

**Q:** How is location transparency achieved and the communication between WLS instances handled?

Inside the Kubernetes cluster, the operator generates a Kubernetes ClusterIP service for each WebLogic server pod called `DOMAINUID-WEBLOGICSERVERNAME` and for each WebLogic cluster called `DOMAINUID-cluster-CLUSTERNAME`. The operator also overrides your WebLogic network listen address configuration to reflect the service names so that you don't need to do this yourself. The services act as DNS names, and allow the pods with WebLogic Servers to move to different nodes in the Kubernetes cluster and continue communicating with other servers.  

**Q:** How is communication from outside the Kubernetes cluster handled?

HTTP communication to your applications from locations outside the cluster is typically accomplished by deploying a load balancer that redirects traffic to your domain's Kubernetes services (the operator automatically deploys these services for you). For example, [link to sample that sets up Traefik+ingress to a cluster - such as quick  start.]

RMI communication with locations outside of the Kubernetes cluster is typically accomplished by deploying a load balancer or using Kubernetes Node Ports, see [external client FAQ].

WebLogic console communication is accomplished using either a Kubernetes NodePort service for the WebLogic admin server that the operator can automatically deploy for you [link to doc or sample for doing this], or by configuring a load balancer to redirect console traffic [link to doc or sample for doing this] to your WebLogic admin server.

WebLogic remote console communication is accomplished by using a load balancer, see [link to doc or sample for doing this].

Reply:
@tbarnes-us Not 100% sure about the links you want here:
1. Using either a Kubernetes NodePort service for the WebLogic admin server 
2. Configuring a load balancer to redirect console traffic
For 1, the QS guide, step 7, https://oracle.github.io/weblogic-kubernetes-operator/quickstart/create-domain/
For 2. ???, https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/azure-kubernetes-service/domain-on-pv/#deploy-sample-application

Reply:
Geez - it looks like we have outages here. There's a new chapter for 'remote console' that's somewhat helpful, but nothing for 'WebLogic console' that I can see except buried too deep in the samples.  

For now:
- Access to arbitrary HTTP apps through a LB: point to the 'ingress' chapter.
- Access to the WL console through a LB: maybe point to the MII sample? 
- WebLogic console through a NP: mention 'domain.spec.adminServer.adminService.channels' and point to the domain resource doc 

For the latter two we should have a dedicated chapter just like we now have for the remote console.

Reply:
We'll get those docs in the next round.